code/modules/clothing/suits/armor.dm

```diff
@@ -127,6 +127,7 @@
 	name = "reactive armor"
 	desc = "Doesn't seem to do much for some reason."
 	var/active = 0
+	var/cd = 0//cooldown specific to reactive armor
```

Comment:
Are you sure you can't use obj/item/clothing/var/cooldown? (added in https://github.com/tgstation/-tg-station/commit/39aaea6f283be376c47b03046c2d901ba0922713#diff-c0790df45ca6836b3267c5b4f75ad285R15)